lib/ditto.ts

```diff
@@ -191,6 +218,22 @@ const executeCommand = async (
         ...(options?.lineNumbers ? { lineNumbers: options.lineNumbers } : {}),
       });
     }
+    case "import-components": {
+      if (options.args.length === 0) {
+        console.error("Please provide a file path.");
```

Comment:
I think it makes sense to write this to stderr, but `console.error` is printing a stack trace as well -- maybe `console.info` or `console.warn` would be better?
<img width="1359" alt="image" src="https://github.com/dittowords/cli/assets/19500384/08cd6bd2-34f1-4c15-bf40-dc990dd14930">

Reply:
Nice suggestion!